helm/kubernetes-cluster-prep/templates/tests/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+# Run from dir in which this script resides
+cd "$(dirname "$0")"
+
+TAG=latest
```

Comment:
Do we need `TAG` to be settable via environment variable? e.g.:
```
TAG="${TAG:-latest}"
```
Or are we tagging the image that gets built (e.g. with "edge") outside of this script?

Reply:
This file is now just a helper file, as the build happens in bin/build
https://github.com/cyberark/conjur-authn-k8s-client/blob/master/bin/build#L27
I can remove this file